src/app/reports/reports.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="container page-content">
```

Comment:
Might be nice to add a screenshot of a selected report to see this button.

Reply:
Added.